src/actions/__tests__/cursorDown.ts

```diff
@@ -65,10 +65,16 @@ describe('normal view', () => {
   })
 
   it('work for sorted thoughts', () => {
+    const text = `
+      - a
+        - =sort
+          - Alphabetical
+            - Asc
+        - m
+        - n
+    `
```

Comment:
I'm fine with replacing the `newThought` and `newSubthought` calls with a single `importText`. However, we should start with the thoughts unordered and then sort them in the test. Starting them in sorted order with `=sort` already applied misses this.

```
- a
  - n
  - m
```

However, it also looks like there was a bug in the test already, as it uses `toggleAttribute` directly rather than `toggleSort`. We should switch to `toggleSort`.